lib/msf/core/rpc/v10/rpc_core.rb

```diff
@@ -137,7 +137,8 @@ def rpc_module_stats
       'post'      => framework.stats.num_post,
       'encoders'  => framework.stats.num_encoders,
       'nops'      => framework.stats.num_nops,
-      'payloads'  => framework.stats.num_payloads
+      'payloads'  => framework.stats.num_payloads,
+      'evasions'  => framework.stats.num_evasion
```

Comment:
Actually one small thing, but we should update the RDoc here to mention that evasions can now be returned and explain this properly.

Reply:
I have updated the api descriptions to mention that the statistics on the evasion modules are now being returned as well. 
I'm not sure you meant this, if not, could you please explain more specifically what I should update?